app/src/app/app-shell/assets/shell.plugin.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../../../../../extension.schema.json",
+  "$id": "14157f82-6489-4187-9b68-1a5e38fadeca",
+  "$name": "About Plugin",
+  "$version": "0.0.1",
+  "$vendor": "Alfresco Software, Ltd.",
+  "$license": "LGPL-3.0",
+
+  "routes": [
+    {
+      "id": "app.shell.login",
+      "path": "login",
+      "auth": ["app.shell.dummy.guard"],
+      "component": "app.shell.login",
+      "layout": "",
+      "data": {
+        "title": "APP.SIGN_IN"
+      }
+    },
+    {
+      "id": "app.shell.blank",
```

Comment:
is this entry needed?

Reply:
I think so, app shell provides right now two pages, the layout and the blank page. 

I extracted most generic one to shell (nevertheless if we consider that this is not needed, we can remove it)

Reply:
Removed as discussed